.github/workflows/test_and_build.yml

```diff
@@ -0,0 +1,54 @@
+name: test_and_build
+on: [push]
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    env:
+      BUNDLE_JOBS: 2
+      BUNDLE_RETRY: 3
+      LANG: en_US.UTF-8
+      PGHOST: postgres
+      PGUSER: vets-api
+      RAILS_ENV: test
+      "Settings.saml.cert_path": "spec/support/certificates/ruby-saml.crt"
+      "Settings.saml.key_path": "spec/support/certificates/ruby-saml.key"
+      "Settings.binaries.clamdscan": "clamscan"
+
+    services:
+      postgres:
+        image: postgis/postgis:11-2.5
+        env:
+          POSTGRES_USER: vets-api
+          POSTGRES_DB: vets-api_test
+          POSTGRES_PASSWORD: postgres
+        ports:
+          - 5432:5432
+
+      redis:
+        image: redis:3.2-alpine
+        ports: ['6379:6379']
+
+
+    steps:
+      - uses: actions/checkout@v2 #checkout step
+      - uses: ruby/setup-ruby@v1      
+      - name: Install dependencies (pdftk, poppler, imagemagick)
+        run: sudo apt install imagemagick pdftk poppler-utils
+
+      - name: Gem Cache
+        uses: actions/cache@v2
+        with:
+          path: vendor/bundle
+          key: gem-cache-${{ matrix.image }}-${{ hashFiles('Gemfile.lock') }}
+
+      - name: Install Dependencies
+        run: bundle install --path vendor/bundle
+  
+      - name: Audit bundled gems for CVE issues
+        run:  bundle exec bundle-audit check --update
+        
+      - name: Static security analysis using Brakeman
+        run:  bundle exec brakeman --exit-on-warn --run-all-checks --confidence-level=2
```

Comment:
* I believe we should be using `--ensure-latest` here.
* `--exit-on-warn` is the default so that's redundant.
* Is `--confidence-level` the right flag for this? I'm not seeing it in their docs.

Reply:
This syntax is what we use for circle, it can definitely be edited